src/AppContainer.tsx

```diff
@@ -146,7 +136,7 @@ const AppContainer: FunctionComponent<Props> = ({ initialLoadComplete, workspace
             <div className={"AppContent"} style={{padding: 0, position: 'absolute', top: appBarHeight, height: H, left: hMargin, width: W, overflowY: 'auto', overflowX: 'hidden'}}>
                 {
                     initialLoadComplete ? (
-                        <Routes width={W} height={H} />
+                        <Routes width={W} height={H} workspaceInfo={workspaceInfo} />
```

Comment:
workspaceInfo is now passed explicitly as a prop rather than part of the global redux state.